_posts/2023-01-27-re-invent-recap-ml.markdown

```diff
@@ -0,0 +1,165 @@
+date:   2023-01-27 04:48:58 +0000
+tags: re:invent2022 MachineLearning sagemaker aws data dataScience MLOps
+author: Nadia Reyhani
+image: /img/reinventRecap2022-ML/reinventBanner.jpg
+---
+
+# What's AWS Re:invent?
+
+[AWS Re:Invent](https://reinvent.awsevents.com/) is the biggest tech conference and cloud event of the year. It's billed as "The most transformative event in tech", a learning conference hosted by AWS where the global cloud community come together to meet, learn and get inspired! 
+AWS Re:invent event has a lot to offer in terms of different tracks and area of interests to follow related to your job domain. Such as Machine learning/AI, storage, serverless, etc,...
+As well as variety of session types i.e workshops to chalktalks with different levels from level-100 (Foundational) to level-400 as expert level! 
+
+Well, if you have not heard about AWS Re:invent or if you couldn't attend it for any reason, that means you have probably missed the AWS Re:invent 2022. That's ok! Good thing I like about Tech world is that there are always people who have attended the events, meet-ups and conferences and most of them are always willing to share their experience including their tech take away from the event. 
+
+This year for the very first time, I attended the AWS Re:invent conference through AWS [All Builders Welcome grant](https://reinvent.awsevents.com/community/all-builders-welcome/). As a machine learning enthusiasm I could participate a few chalktalks in ML/AI learning track as well as Re:invent Announcement sessions. 
+
+In this blog post, I would like to share highlights of some of the cool stuff that are now available or still in preview for Machine learning and data scientist within AWS AI/ML services and more specifically Amazon SageMaker.
+
+To start with, let's look at the end to end workflow of building a Machine Learning solution on AWS and then walk through the new announcements for some of the components in AWS Machine learning architecture. 
+
+
+
+# AWS ML infrastructure and Services
+![AWS Machine Learning Stack](/img/reinventRecap2022-ML/aws-ml-stack.png)
+
+## ML Compute
+As the graphic illustrates, the bottom layer contains the compute, networking and storage services that construct the foundational block of ML infrastructure on AWS. 
+Here are some recently added capabilities to Compute component in Machine Learning solutions on AWS.
+
+### Amazon EC2 inf2 Instances (Preview)
+As per AWS documentations, Inf2 instances are powered by up to 12 AWS Inferentia2, the third AWS-designed DL accelerator. Inf2 instances offer 3x higher compute performance, up to 4x higher throughput, and up to 10x lower latency compared to Inf1 instances. The Inf2 instances are designed to deliver higher performance at the lowest cost for Deep Learning inference applications. You can now efficiently deploy a 175B parameter model for inference across multiple accelerators on a single Inf2 instance.  
```

Comment:
I love the blog Nadia ! 
175B is massive number of parameters - did they talk about the cost of training at all?

Reply:
No in specific I guess hence it's still in preview, user can request for usage and price inquiries, however it's been said that it delivers a better price performance than Inf1 for smaller models!